### PR TITLE
Crowdin integration

### PR DESCRIPTION
**Pull request by ydrisR**
Add what is required to use Crowdin live preview integration

### CODE REVIEW

File: buildconfig.js [hunk 1]
```diff
@@ -76,6 +77,24 @@ if (typeof process.env.MATOMO_TRACKER_URL === 'string' && process.env.MATOMO_TRA
     matomoJs = '/* MaToMo tracker is disabled */';
 }
 
+// Replace Crowdin Javascript
+if (typeof process.env.CROWDIN === 'string' && process.env.CROWDIN === 'enable') {
```
[the remark this answers is not in the document]
Comment (BigFab) [reply buildconfig.js:81]: maybe it is safer to ensure it is set to "enable"? (still just thoughts :) )

File: app/core/language.js [hunk 2]
```diff
@@ -9,64 +9,88 @@ import mainFR from '../../locales/fr/translation.json';
...
-i18n.on('languageChanged', lng => {
-  moment.locale(lng);
-  if (self.localStorage) {
-    self.localStorage.lang = lng;
-  }
-});
+const crowdinActive = typeof _jipt === 'object';
 
-let defaultLanguage = getLocale();
+let language = getLocale();
```
Comment (BigFab) [inline app/core/language.js:16]: would it be more efficient to have this one as an _else_ condition of the below?

File: app/core/language.js [hunk 3]
```diff
@@ -9,64 +9,88 @@ import mainFR from '../../locales/fr/translation.json';
...
+if (crowdinActive) {
+  i18nOptions.fallbackLng = 'it';
+  i18nOptions.resources.it = {
+    main: mainIT,
+    params: parameterEN
+  }
+
+}
+
+// Update moment with the right language, for date display
```
Comment (BigFab) [inline app/core/language.js:77]: these are changes for Timezones PRs we are doing in //?
Shall we rebase the branch?

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,7 +2,9 @@
 Blip is the web front end for YourLoops system.
 It is based on Tidepool Blip 1.27.
 
-## Unreleased
+## 0.13.1 - 2020-03-20
```
Comment (BigFab) [inline CHANGELOG.md:5]: this should be changed
back to "unreleased" until we make a release PR?

File: buildconfig.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (ydrisR) [reply buildconfig.js:81]: I prefer to compare to "enable". !=disable means it can be anything else...

File: app/core/language.js (hunk 2 above)
Comment (ydrisR) [reply app/core/language.js:16]: no

File: CHANGELOG.md (hunk 4 above)
Comment (ydrisR) [reply CHANGELOG.md]: ok

File: config/local.sh [hunk 5]
```diff
@@ -16,3 +16,4 @@ export MATOMO_TRACKER_URL='disable'
 export MAX_FAILED_LOGIN_ATTEMPTS='5'
 export DELAY_BEFORE_NEXT_LOGIN_ATTEMPT='10'
 export TERMS_PRIVACY_DATE='2020-02-26'
+export CROWDIN='disable'
```
Comment (yvon-dblg) [inline config/local.sh:19]:
```suggestion
export CROWDIN='disabled'
```
I know other var is "disable" (it was my mistake), but I see everywhere "disabled" for this situation.
Should we set it in the past tense ?

File: app/core/language.js [hunk 6]
```diff
@@ -9,64 +9,88 @@ import mainFR from '../../locales/fr/translation.json';
...
+      params: parameterDE
+    }
+  }
+}
+
+
+i18n.use(reactI18nextModule);
+
+if (crowdinActive) {
+  i18nOptions.fallbackLng = 'it';
```
Comment (yvon-dblg) [inline app/core/language.js:69]: Why use Italian as a default language ?

File: app/core/language.js [hunk 7]
```diff
@@ -9,64 +9,88 @@ import mainFR from '../../locales/fr/translation.json';
...
+  i18nOptions.resources.it = {
+    main: mainIT,
+    params: parameterEN
```
Comment (yvon-dblg) [inline app/core/language.js:72]: With English parameters?

File: app/core/language.js (hunk 3 above)
Comment (yvon-dblg) [reply app/core/language.js:77]: > 
> 
> these are changes for Timezones PRs we are doing in //?
> Shall we rebase the branch?

No this is the translations for moment.
Ex: to not print January but Janvier in lang is French.

File: locales/translation.json [hunk 8]
```diff
@@ -0,0 +1,693 @@
+{
```
Comment (yvon-dblg) [inline locales/translation.json:1]: Why this file ?

File: config.server.js [hunk 9]
```diff
@@ -58,10 +58,16 @@ if (process.env.SKIP_HAKKEN) {
...
 }
 
-if (typeof process.env.MATOMO_TRACKER_URL === 'string' && process.env.MATOMO_TRACKER_URL !== 'disabled') {
+if (typeof process.env.MATOMO_TRACKER_URL === 'string' && process.env.MATOMO_TRACKER_URL !== 'disable') {
   config.matomoUrl = process.env.MATOMO_TRACKER_URL;
 } else {
   config.matomoUrl = null;
 }
 
+if (typeof process.env.CROWDIN === 'string' && process.env.CROWDIN !== 'disable') {
```
Comment (yvon-dblg) [inline config.server.js:67]: Why test here for disabled and not enabled like above?

File: app/core/language.js (hunk 6 above)
Comment (ydrisR) [reply app/core/language.js:69]: italian is a fake language used for crowdin live preview ;)

File: locales/translation.json (hunk 8 above)
Comment (ydrisR) [reply locales/translation.json:1]: this is the source file for blip strings.
language files (en, fr, de...) will be generated by crowdin an we should not modify them

**Pull request merged**